src/main/java/com/JSONpostroute/validations/ValidationFunctions.java

```diff
@@ -0,0 +1,10 @@
+package com.JSONpostroute.validations;
+
+public class ValidationFunctions {
+    public static boolean reqHasOrdersValidation(String testString){
+        if(testString.contains("orders")){
```

Comment:
It's a good practice use for this condition and return: " return (1 > 0 ? true : false);"

Reply:
You're so right, though intellij suggested an even simpler version after I made the change!